File: apps/ios/Relay/Domain/PanelScan.swift
import Foundation

struct PanelScan: Hashable, Identifiable {
    let id: UUID
    let recognizedControls: [String]

    init(id: UUID = UUID(), recognizedControls: [String]) {
        self.id = id
        self.recognizedControls = recognizedControls
    }

    static let preview = PanelScan(
        recognizedControls: ["Power", "30s", "Start", "Cancel"]
    )
}

File: apps/android/app/src/main/java/com/gaoachao/relay/model/PanelScan.kt
package com.gaoachao.relay.model

import java.util.UUID

data class PanelScan(
    val id: String = UUID.randomUUID().toString(),
    val recognizedControls: List<String>,
) {
    companion object {
        fun demo(): PanelScan = PanelScan(
            recognizedControls = listOf("Power", "30s", "Start", "Cancel"),
        )
    }
}
